server/types/unknown.go

```diff
@@ -93,12 +94,18 @@ func (u UnknownType) GetSerializationID() SerializationID {
 
 // IoInput implements the DoltgresType interface.
 func (u UnknownType) IoInput(ctx *sql.Context, input string) (any, error) {
-	return "", fmt.Errorf("%s cannot receive I/O input", u.String())
+	if strings.ToLower(input) == "null" {
+		return nil, nil
+	}
```

Comment:
This is incorrect. As per the comment on `IoInput` that is on the `DoltgresType` interface:
> An input string will never represent NULL.

If we're receiving the string `"null"` somewhere, then there's an issue somewhere further up the stack that needs to be fixed.